sdk/src/main/java/io/dapr/client/DaprExtensibleClientBuilder.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2021 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io.dapr.client;
+
+import io.grpc.ManagedChannel;
+
+/**
+ * A builder for the DaprClient.
+ */
+public class DaprExtensibleClientBuilder extends DaprClientBuilder{
```

Comment:
I would make this a GRPC specific builder and force the GRPC protocol, even if property is for HTTP.

With the current design, it would be possible for the code to set the GRPC managed channel and a Java property force the client built to be the HTTP instance.

Reply:
Make sense.